docs/administration/managing-users-and-teams/user-roles.md

```diff
@@ -3,45 +3,45 @@ title: User Roles
 description: User Roles are a critical part of the Octopus security model whereby they are assigned to Teams and they dictate what the members of those teams can do in Octopus.
 ---
 
-User Roles play a major part in the Octopus security model. These roles are assigned to Teams and they dictate what the members of those teams can do in Octopus.
+User Roles group permissions and play a major part in the Octopus security model. These roles are assigned to Teams and they dictate what the members of those teams can do in Octopus.
```

Comment:
Sounds funny (grammar) ... "User Roles group permissions `and`" ... Remove `and`?